### ISSUE TITLE
Iphone headers ? 

### ISSUE THREAD

**Issue by rootcstar** [mdesilva/AmazonFlexUnlimited#121]
This code is sending headers for android. Anyone knows the headers for Iphone ?

**Comment by ghost** [comment]
I don't know the exact headers for iPhone. However, I don't believe it's necessary, especially if you are using this repo. When you link your account via the registration link that is generated from this repo, this link generates for you a dummy Android device with a randomly generated serial number (all bots use this same method nowadays). This device is then added to your list of Amazon devices, visible at https: //www. amazon.com/mycd. It's probably best to keep the Android headers since that's what's associated with this dummy device that you created.

Additionally, it doesn't matter if this registration link creates for you an Android device if you use an iPhone for Flex. It's treated as an entirely different device to Amazon, it's like having a second phone. And I doubt Amazon would discriminate against people who have an Android and an iPhone versus people who have 2 iPhones.

**Comment by ghost** [comment]
Actually, I looked back at the code in this script and realized it uses iPhone headers for the most part. It probably should be using Android headers for consistency, but I guess people aren't having issues. The iPhone headers are there in the FlexUnlimited.py file.

**Comment by rootcstar** [comment]
Yes i saw iphone headers too but i was wondering if creating a registration token by an android dummy device causes any problem for later requests with iphone headers.

**Comment by zmkaihan** [comment]
@ssmusic19  please give me an idea of how I can receive the headers "authorization and cookies" dynamically and use that header instead of the static one defined in the FlexUnlimited.py file
